internal/collectors/contracts/erc20/transfer_event.go

```diff
@@ -2,16 +2,16 @@ package erc20
 
 import (
 	"context"
-	"log"
-	"math"
-	"math/big"
-	"sync"
-
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/thepalbi/ethereum-prometheus-exporter/internal/config"
 	"github.com/thepalbi/ethereum-prometheus-exporter/token"
+	"log"
+	"math"
+	"math/big"
+	"sync"
 )
 
 type BlockNumberGetter interface {
```

Comment:
Why are we creating an `interface` with no functionality, just native-type fields? A `struct` seems more appropiate.

Reply:
`type BlockNumberGetter interface {
	BlockNumber(ctx context.Context) (uint64, error)
}`